geospatial/dted.ksy

```diff
@@ -0,0 +1,596 @@
+    - dt2
+  encoding: ASCII
+  endian: be
+  ks-version: 0.8
+  license: MIT
+doc: |
+  PERFORMANCE SPECIFICATION DIGITAL TERRAIN ELEVATION DATA (DTED). |
+  MIL-PRF-89020B (23 May 2000)
+doc-ref: |
+  https://dds.cr.usgs.gov/srtm/version2_1/Documentation/MIL-PDF-89020B.pdf
+seq:
+  - id: uhl
+    type: uhl_header
+    doc: User Header Label (UHL)
+  - id: dsi
+    type: dsi_header
+  - id: acc
+    type: acc_header
+  - id: records
+    type: data_record
+    repeat: expr
+    repeat-expr: uhl.number_of_longitude_lines.value
+types:
+  uhl_header:
+    doc: |
+      User Header Label (UHL). The UHL is used for magnetic tape and CD-ROM. |
+      Fixed Length = 80 ASCII Characters
+    seq:
+      - id: recognition_sentinel
+        contents: UHL
+        doc: |
+          Recognition sentinel
+      - id: fixed_by_standard
+        contents: "1"
+        doc: |
+          Fixed by standard
+      - id: longitude_of_origin
+        type: angle_dddmmssh
+        doc: |
+          Longitude of origin (lower left corner of data set; full degree | 
+          value; leading zero(s) for all subfields: degrees, minutes and |
+          seconds). H is the Hemisphere of the data.
+      - id: latitude_of_origin
+        type: angle_dddmmssh
+        doc: |
+          Latitude of origin (lower left corner of data set; full degree |
+          value; leading zero(s) for all sub fields: (degrees, minutes and |
+          seconds). H is the Hemisphere of the data.
+      - id: longitude_interval_seconds
+        type: ascii_int
+        doc: |
+          Longitude data interval in tenths of seconds.
+      - id: latitude_interval_seconds
+        type: ascii_int
+        doc: |
+          Latitude data interval in tenths of seconds.
+      - id: absolute_vertical_accuracy
+        type: str
+        size: 4
+        doc: |
+          Absolute Vertical Accuracy in Meters. (With 90% assurance that |
+          the linear errors will not exceed this value relative to mean sea |
+          level (Right justified)).
+      - id: security_code
+        type: str
+        size: 3
+        doc: |
+          Security code. (Left Justified)
+      - id: unique_reference
+        type: str
+        size: 12
+        doc: |
+          Unique reference number
+      - id: number_of_longitude_lines
+        type: ascii_int
+        doc: |
+          Count of the number of longitude (profiles) lines for a full |
+          one-degree cell. Count is based on the Level of DTED and the |
+          latitude zone of the cell. (See Table I, II and III).
+      - id: number_of_latitude_points
+        type: ascii_int
+        doc: |
+          Count of the number of latitude points per longitude line for |
+          a full one-degree cell. ( e.g. 1201 for DTED1, 3601.for DTED2).
+      - id: multiple_accuracy
+        type: str
+        size: 1
+        encoding: ASCII
+        doc: |
+          0 - Single |
+          1 - Multiple
+      - id: reserved
+        type: str
```

Comment:
If it is reserved, why str?

Reply:
It is implied by the spec as both the DSI and ACC records specify that the entire section is ASCII.  The UHL record as the spec refers to them is a similar in nature and all fields that have been defined are in ASCII.  The only record type which is different is  the Data Record which follow the metadata inside the UHL, DSI, and ACC records.  However if you recommend that I change it I can change it.